CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+CONTRIBUTING# Contributing
+
+This document provides guidelines for contributing to the module.
+
+## Dependencies
+
+The following dependencies must be installed on the development system:
+
+- [Terraform][terraform-site] v0.12.Z
+- [Docker Engine][docker-engine]
+- [Google Cloud SDK][google-cloud-sdk]
+- [make]
+
+### IAM Roles
```

Comment:
Why is this listed here? These should be listed in the main README.

Reply:
done